BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -540,7 +540,9 @@ - (void)processResponse:(NSURLResponse *)response
 
 - (void)finishURLSessionTaskWithData:(NSData *)data response:(NSURLResponse *)response error:(NSError *)error
 {
-    [self processResponse:response];
+    if (self.HTTPResponse == nil) {
```

Comment:
Right, if the response was already handled, then we don't want to handle it at "finish" time, otherwise it may need to still be handled. I can add a comment.